middleware.cfg

```diff
@@ -40,7 +40,7 @@
 180:file:file
 190:auto:auto
 200:secondary:secondary
-210:etcd:etcd
+210:etcd:github.com/coredns/coredns/middleware/backend/etcdv2
```

Comment:
ugh, because it is one dir deeper, we need the full path? (need to fix that in the generator)